### ISSUE TITLE
POST data to remote URL?

### ISSUE THREAD

**Issue by foo123** [gnuns/allOrigins#101]
Is there any option to post data to the remote page to be fetched?

So: requestMethod = 'POST' along with some payload to be send via POST.

**Comment by akayhanlar** [comment]
I'm wondering that too

**Comment by foo123** [comment]
@akayhanlar 

Looking at the code, it seems there is an option to set the `requestMethod` (via some parameter) but **no option** to include payload/data with the request (except any data set via GET url parameters)

**Comment by skygamer125** [comment]
I really need to know this too...

**Comment by PYUDNG** [comment]
Yep, there is no option to set post data. Although you can send post requests, but cannot send any post body via post request:(

**Comment by foo123** [comment]
Is this going to change in some new update, it would be extra useful with only minimum hassle?

**Comment by PYUDNG** [comment]
I don't know... Actually I agree with your opinion, it must be useful. But I don't know whether the author wants to make it.

**Comment by Inetgeek** [comment]
I'm wondering that too. I can use it to get url raw data, but not post any data to origin url website. If having some solutions, please introducte them, thanks! 👍 💯

**Comment by sherlok235** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
KrystianM28, @ngl756, @bondcoder97, @Kripton99, @gokhandumanofficial, @aurea1123, @OwyJs, @mikedilger, @muratkoca, @nickneymos, @shingonoide, @AzureMan1, @EmreGisi4617, @signalxd, @dan1d, @Kyshman, @PenthosHeydrich, @exequiellol, @coxiff, @manga2

**Comment by SuryaSg** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
Yelchef, @NF-allianz, @moorkedi, @istinyes, @Vladrofl, @aurea1123, @nickody, @klesun, @prolaj, @kiritosan, @weber450, @sunxaio, @haki, @DaisyZhaoo, @bulutgz, @outsider69, @shalai, @SlenderShout, @netvamp, @FursiK911

**Comment by itme3** [comment]
I accept the position.

**Comment by klesun** [comment]
Something is broken with github issues today...
@nickody please fix it when you start working in github =D
And congrats with new position!

**Comment by PYUDNG** [comment]
Well, I need that $180,000 too. As you get that job send me 1/7 of the salary (that is 180,000/7 == $25,714.29 per year, wow delicious!) since I'm one of the 7 participant of this issue XDDD.
And after you start working please get this issue solved as well, I'm eager to use the post feature:)

**Comment by dreamjz** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Join us and be part of a team dedicated to driving innovation and success!

## How to Apply:
To apply for this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.darpal11, @KrebbsTheDeviant, @asdqqvq, @larryto0698, @Yunowds, @cigilypuff, @cyrixhero, @Habdirahman14, @yusufakgunn, @omerfaruknz, @exababy0, @fibtester, @brkylcn35, @mrmehmetsaltan, @Nazariglez, @bobdylan1258215, @Latxosy, @liuweiwei861123, @efcelikk, @cackander
